.docker/opt/configure-aiida.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# This script is executed whenever the docker container is (re)started.
+
+# Debugging.
+set -x
+
+# Environment.
+reentry scan
+export SHELL=/bin/bash
+
+# Setup AiiDA autocompletion.
+grep _VERDI_COMPLETE .bash_profile &> /dev/null || echo 'eval "$(_VERDI_COMPLETE=source verdi)"' >> /home/$SYSTEM_USER/.bashrc
+
+# Check if user requested to set up AiiDA profile (and if it exists already)
+if [[ $SETUP_DEFAULT_PROFILE == true ]] && ! verdi profile show $PROFILE_NAME &> /dev/null; then
+    NEED_SETUP_PROFILE=true;
+else
+    NEED_SETUP_PROFILE=false;
+fi
+
+# Setup AiiDA profile if needed.
+if [[ $NEED_SETUP_PROFILE == true ]]; then
+
+    # Create AiiDA profile.
+    verdi quicksetup                           \
+        --non-interactive                      \
+        --profile $PROFILE_NAME                \
```

Comment:
All these variables need to be wrapped in double quotes in case they contain spaces. Best is to use `"${PROFILE_NAME}"`

Reply:
done